src/stcal/jump/jump.py

```diff
@@ -249,7 +280,12 @@ def detect_jumps(frames_per_group, data, gdq, pdq, err,
             # save the neighbors to be flagged that will be in the next slice
             previous_row_above_gdq = row_above_gdq.copy()
             k += 1
-        elapsed = time.time() - start
+        #  This is the flag that controls the flagging of either snowballs or showers.
+        if expand_large_events:
```

Comment:
i think this block needs to go above so that when multiprocessing is used, the snowball dq flags make it back in to the reconstructed arrays. (i am still trying to verify this is true so i may end up deleting this comment)

Reply:
I'm not clear about what you are proposing. This section has to be after the full dq arrays have been reconstructed. I've done all of my work using multiprocessing. 
The snowball flags are added to the full dq arrays.

Reply:
oh sorry you're right. the snowball flagging doesn't actually use multiprocessing since it's called after